shared/reducers/chat.js

```diff
@@ -45,10 +45,14 @@ function reducer (state: State = initialState, action: Actions) {
 
       const isSelected = state.get('selectedConversation') === action.payload.conversationIDKey
       let snippet
-      try {
-        // $FlowIssue doens't understand try
-        snippet = makeSnippet(message.message.stringValue(), 100)
-      } catch (_) { }
+
+      switch (message.type) {
```

Comment:
After further reflection (and reading more chat code), I'm guessing this is to leave space for other message types in the future. Answered my own question!